### ISSUE TITLE
Is it possible to perform a 2D FFT with gpu.js

### ISSUE THREAD

**Issue by soundnerd** [gpujs/gpu.js#185]
Hello,

I am new to the gpu world.  Is it possible to do a 2D FFT with gpu.js?  Thanks.

Daniel

**Comment by robertleeplummerjr** [comment]
Absolutely.  In-fact: an implementation here https://github.com/dntj/jsfft/blob/master/lib/fft.js#L51 would, with a bit of tlc, likely run fine on gpu.js.

**Comment by Richard-Mathie** [comment]
It's a little difficult as you cannot do inplace operations in webgl, and the threads can not talk to each other. So you will need to do ~`log(n)` kernel calls for each butterfly operation. but it's possible and I would be interested to see if anyone makes it work.

**Comment by fuzzie360** [comment]
As part of reviving the project, we're closing stale support questions and version-specific reports from older releases. A lot changed today: 2.17.0–2.19.2 shipped and are on npm (`npm install gpu.js`), installs work on modern Node, `new GPU()` works in current browsers, Apple Silicon precision issues are fixed, and CI now guards every change. If this is still relevant on gpu.js@2.19.2, please open a fresh issue with a repro — happy to help. (See #807 for the full status update.)
